### PR TITLE
feat(polling): add environment parameters for polling

### PR DESCRIPTION
**Pull request by perrin4869**
small improvement I'd like to have, not breaking (I hope 🤞)

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -1,14 +1,23 @@
+import { env } from "node:process";
+
 /**
  *  @internal
  *  @module Constants
  */
 
+const parseInt = (str: string | undefined): number | undefined => {
+    if (str == undefined) return undefined;
```
Comment (gdp88) [inline src/common/constants.ts:9]: is it intentional we are not using triple equals here? 🤔

**Review by gdp88 — APPROVED** [review]

**Comment by gdp88** [comment]
just a minor comment..lgtm!

File: src/common/constants.ts (hunk 1 above)
Comment (perrin4869) [reply src/common/constants.ts:9]: Oh oops my mistake!! Weird the linter didn't catch it, thanks!

**Comment by perrin4869** [comment]
Ok need to fix this coveralls check lol

**Pull request merged**